Tests/ExportDumpTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+use Cybex\Protector\Exceptions\FailedCreatingDestinationPathException;
+use Cybex\Protector\Exceptions\FailedMysqlCommandException;
+use Cybex\Protector\Exceptions\InvalidConnectionException;
+use Cybex\Protector\Protector;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Storage;
+
+class ExportDumpTest extends BaseTest
+{
+    /**
+     *  Protector instance.
+     */
+    protected Protector $protector;
+
+    protected string $baseDirectory;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        Config::set('protector.baseDirectory', 'protector');
+
+        $this->protector     = app('protector');
+        $this->disk          = Storage::disk('local');
```

Comment:
Could this be done in the BaseTest class, since it is used in every single test?